packages/storybook/stories/va-button-uswds.stories.jsx

```diff
@@ -106,3 +106,57 @@ Disabled.args = {
   disabled: true,
   text: "Disabled",
 };
+
+
+const Template2 = ({
+  back,
+  big,
+   _continue,
+  disableAnalytics,
+  disabled,
+  label,
+  secondary,
+  primaryAlternate,
+  submit,
+  text,
+  messageAriaDescribedby,
+  onsub,
+  onclk,
+}) => {
+  return (
+    <form onSubmit={onsub}>
+      <p>This is inside a form, which has an onsubmit() that displays an alert when the form is submitted</p>
+      <va-button
+        back={back}
+        big={big}
+        continue={_continue}
+        disable-analytics={disableAnalytics}
+        disabled={disabled}
+        label={label}
+        secondary={secondary}
+        primary-alternate={primaryAlternate}
+        submit={submit}
+        text={text}
+        onClick={e => onclk(e) }
+        message-aria-describedby={messageAriaDescribedby}
+      />
+    </form>
+  );
+}
+
+export const Submitted = Template2.bind(null);
+Submitted.args = {
+  ...defaultArgs,
+  onsub : (e)=>{ 
+    // e.preventDefault();
+    console.log(e.target, "I am submitted!!");
+    alert ("form onsubmit method fired!!--on form");
```

Comment:
I think just one exclamation point is sufficient. Also when I submit, the Storybook page rerenders and shows this. Can this be stopped?

![Screenshot 2024-07-23 at 8 43 16 AM](https://github.com/user-attachments/assets/24a5f896-0a87-4903-9486-44b7c43e5751)